docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -76,20 +78,70 @@ AutoGluon automates this process.
 
 AutoGluon automatically and iteratively tests values for hyperparameters to produce the best performance on the validation data. This involves repeatedly training models under different hyperparameter settings and evaluating their performance. This process can be computationally-intensive, so `fit()` can parallelize this process across multiple threads (and machines if distributed resources are available). To control runtimes, you can specify various arguments in fit() as demonstrated in the subsequent **In-Depth** tutorial.
 
-For tabular problems, `fit()` returns a `Predictor` object. Besides inference, this object can also be used to view a summary of what happened during fit.
+For tabular problems, `fit()` returns a `Predictor` object. For classification, you can easily output predicted class probabilities instead of predicted classes:
+
+```{.python .input}
+pred_probs = predictor.predict_proba(test_data_nolab)
+positive_class = [label for label in predictor.class_labels if predictor.class_labels_internal_map[label]==1] # which label is considered 'positive' class
+print(f"Predicted probabilities of class {positive_class}:", pred_probs)
+```
+
+Besides inference, this object can also summarize what happened during fit.
 
 ```{.python .input}
 results = predictor.fit_summary()
 ```
 
-From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can also view what properties AutoGluon automatically inferred about our prediction task:
+From this summary, we can see that AutoGluon trained many different types of models as well as an ensemble of the best-performing models.  The summary also describes the actual models that were trained during fit and how well each model performed on the held-out validation data.  We can view what properties AutoGluon automatically inferred about our prediction task:
 
 ```{.python .input}
 print("AutoGluon infers problem type is: ", predictor.problem_type)
-print("AutoGluon categorized the features as: ", predictor.feature_types)
+print("AutoGluon categorized the features as: ", predictor.feature_types.feature_types_raw)
+special_types = predictor.feature_types.feature_types_special
+if len(special_types) > 0:
+    print("AutoGluon also identified the following special feature types:")
```

Comment:
Missing followup print statement? Does it actually show the special types?

Reply:
replaced all this with:
```
print("AutoGluon identified the following types of features:")
print(predictor.feature_metadata)
```